web/js/containers/map-interactions.js

```diff
@@ -120,6 +52,24 @@ export class MapInteractions extends React.Component {
           </>
 
         )}
+        <OlVectorInteractions
+          changeCursor={changeCursor}
```

Comment:
@jasontk19 I think it's better to lean on the side of connecting more. I believe there are performance benefits (and not to mention pain of prop passing) to using connect instead of prop-passing through numerous levels. In this case I don't think that performance something to consider but it does seem sloppy and feel unnecessary to be prop-checking on both levels when all these props are only relevant to the child. I considered this when isolating this component and skipped it for the sake of time. But, I totally agree with you. I will make map-interactions a folder in the containers area and connect them directly to the child (vector) component. Good call 👍